content/02-understand-prisma/05-under-the-hood.mdx

```diff
@@ -51,3 +51,86 @@ A Prisma engine is the **direct interface to the database**, any higher-level in
 As an example, Prisma Client connects to the [query engine](../reference/tools-and-interfaces/prisma-client/query-engine) in order to read and write data in a database:
 
 ![](https://i.imgur.com/Nwwx5Cv.png)
+
+### Using custom engine binaries
+
+By default, all binaries are automatically downloaded into the `node_modules\@prisma\cli` folder when you install or update `@prisma\cli`. You might want to use a custom binary file if:
+
+* Automated download of binaries is not possible.
+* You have created your own engine binary (for testing, or for an OS that is not officially supported).
+
+Use the following environment variables to specify custom locations for your binaries:
+
+* `PRISMA_MIGRATION_ENGINE_BINARY` (Migration engine)
+* `PRISMA_QUERY_ENGINE_BINARY` (Query engine)
+* `PRISMA_INTROSPECTION_ENGINE_BINARY` (Introspection engine)
+* `PRISMA_FMT_BINARY` (for `npx prisma format`)
+
+
+#### The `prisma\.env` file
+
+To set an environment variable in the `prisma\.env` file:
+
+1. Add the environment variable to the `prisma\.env` file:
+
+   ```
+   PRISMA_QUERY_ENGINE_BINARY=c:\custom\my-query-engine-windows.exe
```

Comment:
```suggestion
   PRISMA_QUERY_ENGINE_BINARY=custom/path/my-query-engine-binary
```

Reply:
I adapted this as most people reading this will most probably not be Windows users.